### ISSUE TITLE
Typing Issues with multiple components

### ISSUE THREAD

**Issue by codyaverett** [reakit/reakit#245]
# Bug report

**What is the current behavior?**

In a pure Typescript Project, Button props aren't inferring correctly. 

Error output
```
[at-loader] Checking finished with 1 errors
[at-loader] ./src/components/Atoms/Button/Button.stories.tsx:7:14
    TS2322: Type '{ children: string; }' is not assignable to type 'Pick<ThemedOuterStyledProps<Pick<Pick<ThemedOuterStyledProps<Pick<{ as: ComponentType<{}>; }, "as"> & { theme?: any; } & BoxProps, any>, "opaque" | "fixed" | "absolute" | "relative" | "static" | "sticky" | "theme" | "palette" | "tone" | "innerRef"> & Pick<...> & { ...; } & StandardProperties<...> & Pick<...>, "as" |...'.
  Property 'boxAlign' is missing in type '{ children: string; }'.
```

**If the current behavior is a bug, please provide the steps to reproduce.**

Just using the component as documented.

![image](https://user-images.githubusercontent.com/3270702/45852853-b9893d80-bd07-11e8-9a7e-29c9e2339e38.png)

**What is the expected behavior?**

I should be able to import button and use it without any required props.

e.g.

```<Button>Text</Button>```

**Other relevant information:**

reakit version:  `0.15.1`

typescript version: `3.0.3`

**Comment by codyaverett** [comment]
From what I have found thus-far it happens for Heading, Card.Fit, and Paragraph as well.  The commonality between these is is that they all expecting me to provide boxProps to the component props. They aren't optional apparently.

Card and Box work fine.

How the typings work in Reakit are a little confusing to me still, so i can't wait to see the results of #229

I'll see what else I can find out.

**Comment by diegohaz** [comment]
Thank you for reporting it, @codyaverett.

I tried to reproduce it with our `test.tsx` without success. I'm gonna try it with another project. How is your tsconfig.json?

**Comment by diegohaz** [comment]
Should be fixed in `v0.15.2`

**Comment by codyaverett** [comment]
Perfect! I tested it and it works now!
Thanks for the quick response and help @diegohaz
